vlasovsolver/vlasovmover.cpp

```diff
@@ -243,6 +240,77 @@ void calculateSpatialTranslation(
    // bailout(true, "", __FILE__, __LINE__);
 }
 
+/** Propagates the distribution function in spatial space.
+    Now does extra calculations locally without interim MPI communication.
+
+    Based on SLICE-3D algorithm: Zerroukat, M., and T. Allen. "A
+    three‐dimensional monotone and conservative semi‐Lagrangian scheme
+    (SLICE‐3D) for transport problems." Quarterly Journal of the Royal
+    Meteorological Society 138.667 (2012): 1640-1651.
+
+ */
+void calculateSpatialLocalTranslation(
+   dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
+   const vector<CellID>& local_propagated_cells, // Used for loadbalancing, not selecting active cells
+   const vector<CellID>& remoteTargetCellsx, // Not necessary, passed for compatibility
+   const vector<CellID>& remoteTargetCellsy, // Not necessary, passed for compatibility
+   const vector<CellID>& remoteTargetCellsz, // Not necessary, passed for compatibility
+   vector<uint>& nPencils,
+   creal dt,
+   const uint popID,
+   Real &time
+   ) {
+
+   int trans_timer;
+   bool AMRtranslationActive = false;
```

Comment:
This guy seems to be always false, in both versions, so it could be just put straight as false at line 275 (and in the previous function too), no?

Reply:
This is due to https://github.com/fmihpc/vlasiator/pull/571
The idea is still to return to this and fix the generation of comm flags at curved boundary regions, when I have time.

Reply:
ok